src/main/app/src/app/zaken/besluit-view/besluit-view.component.ts

```diff
@@ -25,6 +27,7 @@ export class BesluitViewComponent implements OnInit, OnChanges {
     histories: Record<string, MatTableDataSource<HistorieRegel>> = {};
 
     besluitInformatieobjecten: Record<string, DocumentenLijstFormField> = {};
+    toolTipIcon = new TextIcon(Conditionals.always, 'info_outline', 'toolTip_icon', '', 'pointer');
```

Comment:
Volgens mij bestaat info outline niet meer, denk dat het zo moet zijn
toolTipIcon = new TextIcon(Conditionals.always, 'info', 'toolTip_icon', '', 'pointer', true);

Misschien zou je ook nog een Conditionals.hasValue kunnen maken

Reply:
Ah ja, die is net een dag voor de PR gewijzigd - had m'n base branch waarschijnlijk nog niet up-to-date. Is opgelost in opvolgende commit.